filters/diag/diag.go

```diff
@@ -108,7 +110,16 @@ type jitter struct {
 	sleep func(time.Duration)
 }
 
-var randomChars = []byte("ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789")
+func genCharArray(s string) [62]byte {
+	res := [62]byte{}
+	for i := 0; i < len(s); i++ {
+		res[i] = s[i]
+	}
+
+	return res
+}
+
+var randomChars = genCharArray("ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789")
```

Comment:
In Go arrays can be concurrently accessed without lock, slices can grow so they can not access concurrently